File: LAB2/2024-1/LAB2-2024_1-Desarrollo/Consultorio.java
public class Consultorio extends AmbienteClinico {
	private String numero;
	
	public Consultorio(double area, char torre, int piso, String numero) {
		super(area, torre, piso);
		this.numero = numero;
	}
	
	@Override
	public String devolverInformacion() {
		return "CONSULTORIO:" + numero + " - TORRE:" + torre + " - PISO:" + piso + "\n";
	}
}

File: LAB4/2024-1/MedicalSoft/MedicalSoftModel/Consultorio.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MedicalSoftModel
{
    public class Consultorio : AmbienteClinico
    {
        private string numero;
        public string Numero { get => numero; set => numero = value; }
        public override void mostrarDatos()
        {
            System.Console.WriteLine("CONSULTORIO:" + numero + " - TORRE:" + Torre + " - PISO:" + Piso);
        }
    }
}
